src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/CanvasView.cs

```diff
@@ -68,18 +72,29 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         {
             Interop.CanvasView.DeleteCanvasView(swigCPtr);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            drawables.Clear();
```

Comment:
Hello, I think this `drawables.Clear();` moves into line 66.
managed resource, `drawables` needs to be freed before native objects be freed.
so it would be =>

        [EditorBrowsable(EditorBrowsableState.Never)]
        protected override void Dispose(DisposeTypes type)
        {
            if (disposed)
            {
                return;
            }
            drawables.Clear();
            base.Dispose(type);
        }

Reply:
to access managed object in Dispose, we only access when disposing is true,
if not, managed object could be already released